src/inputs/index.ts

```diff
@@ -15,7 +15,7 @@ const options: InputOptions = {
 const parseInputPath = (name: string) => expandTilde(getInput(name, options))
 
 export const getInputs = (): Inputs => ({
-  version: getInput('version', options),
+  version: getInput('version'),
```

Comment:
I don't understand this change. Is this intentional? If it is, please explain.

Reply:
Currently, `version` in the action is required. I want to make it optional.